src/coreclr/inc/regdisp.h

```diff
@@ -401,13 +397,10 @@ inline void SyncRegDisplayToCurrentContext(REGDISPLAY* pRD)
 #if defined(TARGET_64BIT)
     pRD->SP         = (INT_PTR)GetSP(pRD->pCurrentContext);
     pRD->ControlPC  = (INT_PTR)GetIP(pRD->pCurrentContext);
```

Comment:
```suggestion
    pRD->SP         = (TADDR)GetSP(pRD->pCurrentContext);
    pRD->ControlPC  = (TADDR)GetIP(pRD->pCurrentContext);
```

Reply:
Do we even need to cast `(TADDR)GetSP(pRD->pCurrentContext);`? It should already return TADDR.